### PR TITLE
fix #1496

### LINKED ISSUE #1496: Broken buttons on YouTube clips
When viewing any clip on YouTube (youtube.com/clips/...) all of the SponsorBlock buttons show up:
image

### PR DESCRIPTION
**Pull request by mchangrh**
- [x] I agree to license my contribution under LGPL-3.0 **or** my contribution is from another project with a license compatible with LGPL-3.0

To test this pull request, follow the [instructions in the wiki](https://github.com/ajayyy/SponsorBlock/wiki/Testing-a-Pull-Request).

***

- force return bad videoID on clip
- don't createbuttons on video ready, updateVisibility with more checks
- don't switch to falsy videoIDs if already available

### CODE REVIEW

**Comment by ajayyy** [comment]
Will this make segments stay there when you move to a non video page such as a channel page?

**Comment by mchangrh** [comment]
> hold on, adding debugging statements to check - it should `resetValues()` back to null

**Comment by mchangrh** [comment]
video -> channel ✅
video -> home ❌
home -> hover video ✅

File: src/content.ts [hunk 1]
```diff
@@ -1218,7 +1220,7 @@ function getYouTubeVideoID(document: Document, url?: string): string | boolean {
     // pageType shortcut
     if (pageType === PageType.Channel) return getYouTubeVideoIDFromDocument()
     // clips should never skip, going from clip to full video has no indications.
-    if (url.includes("youtube.com/clip/")) return false;
+    if (url.includes("youtube.com/clip/")) return "clipNoID";
```
Comment (ajayyy) [inline src/content.ts:1223]: What's the reason for making a string here?

Comment (mchangrh) [reply src/content.ts:1223]: otherwise it'll be overwritten by "false"

**Pull request merged**